CorsixTH/Luatest/spec/entities/humanoids/vip_spec.lua

```diff
@@ -41,34 +41,34 @@ local function create_room(name, args)
 
   if args.alive_plants then
     for _=1,args.alive_plants do
-      room.objects[{object_type = {id = "plant"}, isDying = function() return false end}] = true
+      room.objects[{object_type = {id = "plant"}, isMachine = function() return false end, isDying = function() return false end}] = true
     end
   end
 
   if args.dying_plants then
     for _=1,args.dying_plants do
-      room.objects[{object_type = {id = "plant"}, isDying = function() return true end}] = true
+      room.objects[{object_type = {id = "plant"}, isMachine = function() return false end, isDying = function() return true end}] = true
     end
   end
 
   if args.working_machines then
     for _=1,args.working_machines do
-      room.objects[{object_type = {id = "machine"}, strength=_, isBreaking = function() return false end}] = true
+      room.objects[{object_type = {id = "machine"}, strength=_, isMachine = function() return true end, isBreaking = function() return false end}] = true
```

Comment:
```suggestion
      room.objects[{object_type = {id = "machine"}, strength = _, isMachine = function() return true end, isBreaking = function() return false end}] = true
```
Also, the `_` variable is only used for unused place holders. In other words, don't use it. Instead, use a proper variable name in the `for` statement above.

Last but not least, if you split the table literal into several lines, it's simpler to read.

Reply:
Alternatively, perhaps create local variables for the `return true` and `return false` functions, and use those variables instead of duplicating the function definition all the time.

Reply:
It's fine. I just point out things I notice.

Unless there is something truly broken, I usually don't make hard demands for changes.